opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -0,0 +1,51 @@
+import os
+from google.cloud.monitoring import Client
+from google.cloud.monitoring import Metric
+from google.cloud.monitoring import MetricDescriptor
+from google.cloud.monitoring import MetricKind, ValueType
+from google.cloud.monitoring import LabelValueType
+from google.cloud.monitoring import LabelDescriptor
+from opencensus.stats import view
+
+class StackDriverExporter(object):
+    def __init__(self, client=None, project_id=None):
+        if client is None:
+            client = Client(project=project_id)
+
+        self.client = client
+        self.project_id = client.project
+
+    def emit(self, metrics):
```

Comment:
Implement this function by calling the API in google cloud monitoring client to actually send something to stackdriver.